modules/creating-a-project-workbench.adoc

```diff
@@ -14,51 +14,64 @@ endif::[]
-... From the *Container size* list, select a container size for your server.
-... Optional: From the *Accelerator* list, select an accelerator.
-... If you selected an accelerator in the preceding step, specify the number of accelerators to use.
-.. Optional: Select and specify values for any new *environment variables*.
-////
-ifdef::upstream[]
+.. In the *Notebook image* section, complete the fields to specify the workbench image to use with your workbench.
 +
-[NOTE]
---
-To enable data science pipelines in JupyterLab, create the following environment variable:
-`PIPELINES_SSL_SA_CERTS=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`
---
-endif::[]
-ifdef::self-managed[]
+From the *Image selection* list, select a workbench image that suits your use case. A workbench image includes an IDE and Python packages (reusable code). Optionally, click the *View package information* option to view a list of packages that are included in the image that you selected.
++
+If the workbench image has multiple versions available, select the workbench image version to use from the *Versions* section. To use the latest package versions, Red Hat recommends that you use the most recently added image. 
++
+NOTE: You can change the workbench image after you create the workbench.
+
+.. In the *Deployment size* section, from the *Container size* list, select a container size for your server. The container size controls the number of CPUs, the amount of memory, and the minimum and maximum request capacity of the container. 
+
+.. Optional: Select and specify values for any environment variables. 
++
+Setting environment variables during the workbench configuration helps you save time later because you do not need to define them in the body of your notebooks, or with the IDE command line interface. 
++
+If you are using S3-compatible storage, add these recommended environment variables:
 +
-[NOTE]
 --
-To enable data science pipelines in JupyterLab in self-managed deployments, create the following environment variable:
-`PIPELINES_SSL_SA_CERTS=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`
+* `AWS_ACCESS_KEY_ID` specifies your Access Key ID for Amazon Web Services.
+* `AWS_SECRET_ACCESS_KEY` specifies your Secret access key for the account specified in `AWS_ACCESS_KEY_ID`. 
 --
-endif::[]
-////
-.. Configure the storage for your {productname-short} cluster.
-... Select *Create new persistent storage* to create storage that is retained after you log out of {productname-short}. Complete the relevant fields to define the storage.
-... Select *Use existing persistent storage* to reuse existing storage and select the storage from the *Persistent storage* list.
-.. To use a data connection, in the *Data connections* section, select the *Use a data connection* checkbox.
++
+{productname-short} stores the credentials as Kubernetes secrets in a protected namespace if you select "secret" when you add the variable. 
```

Comment:
For consistency, can capitalize and bold 'secret' instead of using double quotes.